arcade/sprite/base.py

```diff
@@ -57,6 +59,7 @@ def __init__(
         self._width = texture.width * scale
         self._height = texture.height * scale
         self._scale = scale, scale
+        self._visible = visible
```

Comment:
Since we implicitly converted truthy values in the old setter, I'd do it here too.
```suggestion
        self._visible = bool(visible)
```